crates/metrics/src/lib.rs

```diff
@@ -17,19 +26,27 @@ const DEFAULT_SERVICE_NAME: &str = "unknown_service";
 
 static SERVICE_NAME: Mutex<Option<&str>> = Mutex::new(None);
 
+pub type MeterProviderBuilderIntercept = fn(MeterProviderBuilder) -> MeterProviderBuilder;
```

Comment:
I believe the convention is to call it something like `MeterProviderBuilderFn`.

Reply:
Renamed in [bd56a39](https://github.com/WalletConnect/utils-rs/pull/18/commits/bd56a395154758f1082d304b671daba926eae595)